### DISCUSSION TITLE
Not getting approval of Students GitHub Pack

### DISCUSSION THREAD

**Discussion by ISHAAN-KKR** [community/community#131890] [GitHub Education]
### Hi everyone,

I applied two times, it shows the same thing, that I'm not on the location of college. I'm currently in my hometown away from college so I'm applying from here, it saying I'm away from campus. I gave the photo of my college notice regarding the start of semister, but still getting rejected for GitHub student pack.

**Comment by benammarfares** [comment]
You should contact them then in two days you will get your answer

**Comment by samus-aran** [comment]
Hi @ISHAAN-KKR 👋 

The first thing you will need to resolve is setting up 2FA. Did you get a rejection reason for this? Once that is resolved please try again. 👍  Let us know how you get on.
